docs/organization/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -20,14 +20,15 @@ Sentry can track changes to feature flag definitions and report suspicious featu
 
 ### Set Up Change Tracking
 
-Enabling Change Tracking is a two step process. To get started visit the [feature-flags settings page](https://sentry.io/orgredirect/organizations/:orgslug/settings/feature-flags/) in a new tab. Then follow the steps listed below.
+Enabling Change Tracking is a three step process. To get started visit the [feature-flags settings page](https://sentry.io/orgredirect/organizations/:orgslug/settings/feature-flags) in a new tab. Then follow the steps listed below.
 
-1. **Register the webhook URL**.
-    - Under the "Integrate Feature Flag Service" heading a Sentry webhook URL has been provided.
-    - Copy the Sentry webhook URL revealed in the fly out and paste it into LaunchDarkly within their [webhook integration UI](https://app.launchdarkly.com/settings/integrations/webhooks/new).
-2. **Set the Signing Secret**.
+1. **Click the "Add New Provider" button.
+    - One webhook secret can be registered per provider type.
+2. **Register the webhook URL**.
```

Comment:
Where do we get this URL?

Reply:
Its on the page (or rather will be after michelle makes a change).

Reply:
On the "Add New Provider" form page.

Reply:
Ok, could we have it on both form states? (Before/after submitting secret) cc @michellewzhang